src/vsr/replica.zig

```diff
@@ -2905,6 +2916,32 @@ pub fn ReplicaType(
             }
         }
 
+        fn on_pulse_timeout(self: *Self) void {
+            if (constants.aof_recovery) unreachable;
```

Comment:
Why not `comptime assert(!constants.aof_recovery);`?

Reply:
I wanted it to be a runtime panic; otherwise, I would need to comptime disable the timer logic as well.